railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -90,5 +90,8 @@ Rails.application.configure do
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # Don't mount Action Cable in the main server process.
+  # config.action_cable.mount_path = nil
```

Comment:
If someone don't want to mount Action Cable in the main server they would need `url` too, so it is better to show it here.